oak_runtime/src/channel.rs

```diff
@@ -121,6 +121,12 @@ impl ChannelHalf {
         self.channel.id
     }
 
+    /// Get the ID of the underlying channel.  For debugging/introspection
+    /// purposes.
+    pub fn get_channel_name(&self) -> String {
```

Comment:
nit: Does this need to be a `String`? Would `&str` not be sufficient?

Reply:
good point, will make it that